### DISCUSSION TITLE
Is it possible to create my own custom "Emoji Font" using fonttools?

### DISCUSSION THREAD

**Discussion by ziontee113** [fonttools/fonttools#2703] [Q&A]
Hi, I'm trying to have some fully colored images into my Terminal as characters. So Nerd Fonts is not an option since it can't support characters, but most terminal emulators support Emojis, which are quite detailed & fully colored.

I wonder if I learn python and fonttools, I can finally achieve that dream of having :kekw in my comments while coding in my terminal.

Thank you for reading :)

**Comment by ChiefMikeK** [comment]
Try to open your patched Nerd-font and edit it with fontforge.  
test with different formatted outputs / tables(some tables still are WIP).  
And also you might create a repo(or fork) on your github account and provide a link here?

https://fontforge.org/docs/ui/dialogs/multilayer.html

**Comment by miguelsousa** [comment]
@ziontee113 have a look at the contents of this repo: https://github.com/adobe-fonts/noto-emoji-svg
In particular, [make_bw_font.py](https://github.com/adobe-fonts/noto-emoji-svg/blob/main/make_bw_font.py) uses FontTools to produce a regular (sans color) font from black & white SVG files.
Then [make_svg_font.py](https://github.com/adobe-fonts/noto-emoji-svg/blob/main/make_svg_font.py) uses FontTools to add an `SVG ` table (containing the colored SVG artwork) to the B&W font.
